composer.json

```diff
@@ -32,7 +32,7 @@
         "phpunit/phpunit": "^8.5.2",
         "roave/no-leaks": "^1.1",
         "squizlabs/php_codesniffer": "^3.5.4",
-        "vimeo/psalm": "^3.8"
+        "vimeo/psalm": "^3.9"
```

Comment:
Please also update `composer.lock`: this project does use that :)

Reply:
Somehow, the locked version is already 3.9.3

Reply:
Yeah, but the hash in `composer.lock` should still change

Reply:
Oh. Never used that before. After a quick search, I found composer update --lock. Is that it? It returns me a weird error (This package requires php ^7.4 but your PHP version (7.3.99; Package overridden via config.platform (actual: 7.4.1)) does not satisfy that requirement.)

Reply:
Ow, we really need to drop `config.platform` then :S 

Can you try that?

Reply:
It worked for regenerating the lock. Should I also commit the composer.json without platform?